### DISCUSSION TITLE
Can't get APU started and no standby HSI

### DISCUSSION THREAD

**Discussion by FSNut01** [flybywiresim/aircraft#3045] [Help]
For the past few days I have not been able to get the APU started. It was working fine until I had a sudden CTD a few nights ago while in flight with another aircraft. Everything else seems to be working fine (MCDU, FCP, PFD's, switches etc.) but no APU. The switches light up but nothing happens. The only other mods I have installed besides numerous liveries are the WT CJ4 and their 1000 and 3000 mods. MSFS is up to date and I have the most recent dev build of the A320 installed. 

In addition for several weeks I have not been able to get anything on the standby HSI screen. Its' blank. 

Any thoughts on either? Steps to take?

**Comment by tracernz** [comment]
Try remove A32NX and see if you have the same issue with the base A320 from Asobo? At least then you have a better idea where the issue lies.

**Reply by FSNut01** [reply]
I did as you suggested and the default A320 displayed the same behavior. So I made a backup of my Community folder, deleted everything and re-installed the FBW A32NX and both problems have been resolved for now. I'll start adding back in the mods one by one to see if I can identify the one that breaks it. As stated above the only mods I had were the Working Title mods for the CJ4 and GS1000/3000s. I also had a bunch of liveries.
